db/migrate/20231128100004_add_post_counter.rb

```diff
@@ -0,0 +1,5 @@
+class AddPostCounter < ActiveRecord::Migration[7.1]
+  def change
+    add_column :users, :post_counter, :integer
+  end
```

Comment:
- [ ] Great job fixing the issues mentioned by previous reviewer, but there's one issue. It should be `posts_counter` instead of `post_counter`, It's a project requirements to follow the column names from [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png). Please make sure to fix this issue.